docs/semgrep-supply-chain/overview.md

```diff
@@ -21,121 +21,112 @@ Object.entries(frontMatter).filter(
 }
 </ul>
 
-# Conceptual overview of Semgrep Supply Chain
-
 <SscIntro />
 
-![Semgrep Supply chain Vulnerabilities page](/img/sc-vulns.png)
+![Semgrep Supply Chain Vulnerabilities page](/img/sc-vulns.png)
 _Figure 1_. Semgrep Supply Chain Vulnerabilities page.
 
-Semgrep Supply Chain parses lock files for a list of dependencies, then scans your codebase using **rules **written with Semgrep's pattern syntax. Supply Chain rules specify the following:
+## Identify open source security vulnerabilities
+
+Semgrep Supply Chain detects [security
+vulnerabilities](https://nvd.nist.gov/vuln/full-listing) in your codebase
+introduced by open-source dependencies, using high-signal rules to determine the
+vulnerability's [reachability](/semgrep-supply-chain/glossary/#reachability).
+
+To do this, Semgrep Supply Chain parses lockfiles for a list of dependencies,
+then scans your codebase using [rules](/running-rules/) that specify the
+following information:
 
-* The range of versions that contain the dependency's vulnerability.
-* A pattern for vulnerable code, such as passing in unsanitized data.
+* The dependency versions that contain a vulnerability;
+* The pattern for the vulnerable code that Semgrep compares against your code;
 * The severity of the vulnerability.
 
-Semgrep Supply Chain generates a **finding** when it detects a match. If the dependency's version is within the range and finds the matching code within your codebase, the finding is **reachable**.
+The following diagram shows the relationship between a Semgrep Supply Chain
+rule, the codebase scanned, and its lockfile:
 
-A finding is **unreachable** if the dependency contains a known vulnerability, but the vulnerable matching code is not used in your codebase.
+![Relationship between a Semgrep Supply Chain rule, lockfile, CVE record, and codebase](/img/sc-reachability-analysis.png)
+_Figure 2_. Relationship between a Supply Chain rule, lockfile, CVE record, and codebase.
 
-A finding is **undetermined** if Semgrep detects a match for the dependency in the vulnerable version range, but does not have a reachability rule for this vulnerability. This is most common with older or lower severity vulnerabilities.
+### Types of Semgrep Supply Chain findings
 
-Undetermined findings are generated by Semgrep Supply Chain **[lockfile-only](/semgrep-supply-chain/glossary/#lockfile-only-rules)** rules, which are rules that do not perform reachability analysis. These findings are generated by checking the dependency's version in its lock or manifest file against versions with known vulnerabilities. These rules are on par with [GitHub Dependabot](https://github.com/dependabot).
```

Comment:
Curious why we removed the dependabot mention?

Reply:
I would be happy to return the original content, but it seemed a bit weird to mention Dependabot as a standard?